### PR TITLE
Release/v5.5.6: Changelog

### PR DESCRIPTION
**Pull request by ashnwade**
This PR addresses no issue. 
Changelog for the v5.5.6 release.

### CODE REVIEW

File: changelog/5.5.6.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Released 07 October 2024
+
+## Gravwell
+
+```{note}
+This release contains high priority bug fixes for multifiltering acceleration.
+Gravwell highly recommends that all users upgrade to 5.5.6 or above.
+
+False negatives may occur in versions 5.3.0-5.5.5 under the following conditions:
```
Comment (david-fritz-gravwell) [inline changelog/5.5.6.md:11]:
```suggestion
False negatives may occur in versions 5.3.0-5.5.5 when using multifilters with the index engine combined with the fulltext accelerator.
```

File: changelog/5.5.6.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Using a multifilter where one of the terms does not meet the indexing rules (min length, etc)
```
Comment (david-fritz-gravwell) [inline changelog/5.5.6.md:12]: I'd remove these bullets in favor of just the sentence change above.

**Review by david-fritz-gravwell — CHANGES_REQUESTED** [review]

**Comment by ashnwade** [comment]
@david-fritz-gravwell PTAL

**Review by david-fritz-gravwell — APPROVED** [review]
LGTM

**Pull request merged**